app/src/main/java/org/oppia/android/app/home/recentlyplayed/OngoingListAdapter.kt

```diff
@@ -61,104 +58,7 @@ class OngoingListAdapter(
-              (activity as Context).resources
-                .getDimensionPixelSize(R.dimen.ongoing_story_card_view_margin_top_portrait)
-            } else {
-              (activity as Context).resources
-                .getDimensionPixelSize(
-                  R.dimen.ongoing_story_card_view_margin_top_portrait_story_grid_greater_then_2
-                )
-            }
-          } else {
-            (activity as Context).resources
-              .getDimensionPixelSize(R.dimen.ongoing_story_card_view_margin_top_landscape)
-          }
-        }
-        val marginBottom = 0
-        when (spanCount) {
-          2 -> {
-            when {
-              storyGridPosition % spanCount == 0 -> params.setMargins(
-                marginMin,
-                marginTop,
-                marginMax,
-                marginBottom
-              )
-              else -> params.setMargins(
-                marginMax,
-                marginTop,
-                marginMin,
-                marginBottom
-              )
-            }
-          }
-          3 -> {
-            when {
-              storyGridPosition % spanCount == 1 -> params.setMargins(
-                marginMax,
-                marginTop,
-                /* right= */ 0,
-                marginBottom
-              )
-              storyGridPosition % spanCount == 2 -> params.setMargins(
-                marginMin,
-                marginTop,
-                marginMin,
-                marginBottom
-              )
-              storyGridPosition % spanCount == 0 -> params.setMargins(
-                /* left= */ 0,
-                marginTop,
-                marginMax,
-                marginBottom
-              )
-            }
-          }
-          4 -> {
-            when {
-              (storyGridPosition) % spanCount == 1 -> params.setMargins(
-                marginMax,
-                marginTop,
-                /* right= */ 0,
-                marginBottom
-              )
-              (storyGridPosition) % spanCount == 2 -> params.setMargins(
-                marginMin,
-                marginTop,
-                marginMin / 2,
-                marginBottom
-              )
-              (storyGridPosition) % spanCount == 3 -> params.setMargins(
-                marginMin / 2,
-                marginTop,
-                marginMin,
-                marginBottom
-              )
-              (storyGridPosition) % spanCount == 0 -> params.setMargins(
-                /* left= */ 0,
-                marginTop,
-                marginMax,
-                marginBottom
-              )
-            }
-          }
-        }
-        holder.binding.ongoingStoryCardView.layoutParams = params
-        holder.binding.ongoingStoryCardView.requestLayout()
-      }
-      else -> throw IllegalArgumentException("Invalid item view type: ${holder.itemViewType}")
+      }else -> throw IllegalArgumentException("Invalid item view type: ${holder.itemViewType}")
```

Comment:
Nit: missing space between the brace & ``else``.

Reply:
Done.